src/exercises/exerciseTypes/findWordInContextCloze/FindWordInContextCloze.js

```diff
@@ -0,0 +1,152 @@
+import BottomInput from "../findWordInContext/BottomInput.js";
+
+const EXERCISE_TYPE = "Cloze_L1_to_L2";
+export default function FindWordInContextCloze({
+  api,
+  bookmarksToStudy,
+  correctAnswer,
+  notifyIncorrectAnswer,
+  setExerciseType,
+  isCorrect,
+  setIsCorrect,
+  moveToNextExercise,
+  toggleShow,
+  reload,
+  setReload,
+  exerciseSessionId,
+  activeSessionDuration,
+}) {
+  const [messageToAPI, setMessageToAPI] = useState("");
+  const [articleInfo, setArticleInfo] = useState();
+  const [interactiveText, setInteractiveText] = useState();
+  const speech = useContext(SpeechContext);
+  const [getCurrentSubSessionDuration] = useSubSessionTimer(
+    activeSessionDuration,
+  );
+
+  useEffect(() => {
+    setExerciseType(EXERCISE_TYPE);
+    api.getArticleInfo(bookmarksToStudy[0].article_id, (articleInfo) => {
+      setInteractiveText(
+        new InteractiveText(
+          bookmarksToStudy[0].context,
+          articleInfo,
+          api,
+          "TRANSLATE WORDS IN EXERCISE",
+          EXERCISE_TYPE,
+          speech,
+        ),
+      );
+      setArticleInfo(articleInfo);
+    });
+  }, []);
+
+  function handleShowSolution(e, message) {
+    e.preventDefault();
+    let concatMessage;
+    if (!message) {
+      concatMessage = messageToAPI + "S";
+    } else {
+      concatMessage = message;
+    }
+
+    notifyIncorrectAnswer(bookmarksToStudy[0]);
+    setIsCorrect(true);
+    setMessageToAPI(concatMessage);                        
+    api.uploadExerciseFinalizedData(
+      concatMessage,
+      EXERCISE_TYPE,
+      getCurrentSubSessionDuration(activeSessionDuration, "ms"),
+      bookmarksToStudy[0].id,
+      exerciseSessionId,
+    );
+  }
+
+  function handleCorrectAnswer(message) {
+    setMessageToAPI(message);
+    correctAnswer(bookmarksToStudy[0]);
+    setIsCorrect(true);
+    api.uploadExerciseFinalizedData(
+      message,
+      EXERCISE_TYPE,
+      getCurrentSubSessionDuration(activeSessionDuration, "ms"),
+      bookmarksToStudy[0].id,
+      exerciseSessionId,
+    );
+  }
+
+  function handleIncorrectAnswer() {
+    setMessageToAPI(messageToAPI + "W");
+    notifyIncorrectAnswer(bookmarksToStudy[0]);
+  }
+
+  if (!articleInfo) {
+    return <LoadingAnimation />;
+  }
+
+  return (
+    <s.Exercise className="findWordInContextCloze">
+      <div className="headlineWithMoreSpace">
+        {strings.findWordInContextClozeHeadline}
+      </div>
+      
+      {!isCorrect && (
+        <>
+            <h1 className="wordInContextHeadline">{bookmarksToStudy[0].to}</h1>
+            <div className="contextExample">
+                <TranslatableText
+                    isCorrect={isCorrect}
+                    interactiveText={interactiveText}
+                    translating={true}
+                    pronouncing={false}
+                    bookmarkToStudy={bookmarksToStudy[0].from}
+                />
+            </div>
+            <BottomInput
```

Comment:
Why not just render this conditionally on whether it's not correct?

I think this would simplify the code. Essentially, only the BottomInput needs to be checked if the exercise is correct.